README.md

```diff
@@ -6,69 +6,25 @@ This repository is the home of Pulp's CI files.
+Deb | [![Pulp Deb Nightly CI](https://github.com/pulp/pulp_deb/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_deb/actions/workflows/nightly.yml) | [![pULP Deb CI Update](https://github.com/pulp/pulp_deb/actions/workflows/update_ci.yml/badge.svg)](https://github.com/pulp/pulp_deb/actions/workflows/update_ci.yml)
+File | [![Pulp File Nightly CI](https://github.com/pulp/pulp_file/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_file/actions/workflows/nightly.yml) | [![Pulp File CI Update](https://github.com/pulp/pulp_file/actions/workflows/update_ci.yml/badge.svg)](https://github.com/pulp/pulp_file/actions/workflows/update_ci.yml)
+Gem | [![Pulp Gem Nightly CI](https://github.com/pulp/pulp_gem/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_gem/actions/workflows/nightly.yml) | [![Pulp Gem CI Update](https://github.com/pulp/pulp_gem/actions/workflows/update_ci.yml/badge.svg)](https://github.com/pulp/pulp_gem/actions/workflows/update_ci.yml)
+Maven | [![Pulp Maven Nightly CI](https://github.com/pulp/pulp_maven/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_maven/actions/workflows/nightly.yml) | [![Pulp Maven CI Update](https://github.com/pulp/pulp_maven/actions/workflows/update_ci.yml/badge.svg)](https://github.com/pulp/pulp_maven/actions/workflows/update_ci.yml)
+NPM | [![Pulp Npm Nightly CI](https://github.com/pulp/pulp_npm/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_npm/actions/workflows/nightly.yml) | [![Pulp Npm CI Update](https://github.com/pulp/pulp_npm/actions/workflows/update_ci.yml/badge.svg)](https://github.com/pulp/pulp_npm/actions/workflows/update_ci.yml)
+OSTree | [![Pulp OSTree Nightly CI](https://github.com/pulp/pulp_ostree/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_ostree/actions/workflows/nightly.yml) | [![Pulp OSTree CI Update](https://github.com/pulp/pulp_ostree/actions/workflows/update_ci.yml/badge.svg)](https://github.com/pulp/pulp_ostree/actions/workflows/update_ci.yml)
+Python | [![Pulp Python Nightly CI](https://github.com/pulp/pulp_python/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_python/actions/workflows/nightly.yml) | [![Pulp Python CI Update](https://github.com/pulp/pulp_python/actions/workflows/update_ci.yml/badge.svg)](https://github.com/pulp/pulp_python/actions/workflows/update_ci.yml)
+RPM | [![Pulp Rpm Nightly CI](https://github.com/pulp/pulp_rpm/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp_rpm/actions/workflows/nightly.yml) | [![Pulp Rpm CI Update](https://github.com/pulp/pulp_rpm/actions/workflows/update_ci.yml/badge.svg)](https://github.com/pulp/pulp_rpm/actions/workflows/update_ci.yml)
+pulp-cli | [![pulp-cli Nightly](https://github.com/pulp/pulp-cli/actions/workflows/nightly.yml/badge.svg)](https://github.com/pulp/pulp-cli/actions/workflows/nightly.yml) | \*/\*
```

Comment:
*/* looks to me like "something isn't rendering" - can this be "NA", or maybe just "---" instead?